awscli/customizations/s3/results.py

```diff
@@ -326,16 +341,19 @@ def _print_noop(self, **kwargs):
 
     def _print_success(self, result, **kwargs):
         success_statement = self.SUCCESS_FORMAT.format(
-            transfer_type=result.transfer_type, src=result.src,
-            dest=result.dest)
+            transfer_type=result.transfer_type,
+            direction=self._get_direction(result)
```

Comment:
Can we rename `direction`?  "one way direction" sort of implies transfer in one direction, i.e local->s3.  Wasn't obvious to me what that meant.  What about something like `transfer_description`, `transfer_src_dest`, `transfer_locations`, etc.